dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/trackedentity/RequestParams.java

```diff
@@ -63,11 +63,20 @@ class RequestParams extends PagingAndSortingCriteriaAdapter
 {
     static final String DEFAULT_FIELDS_PARAM = "*,!relationships,!enrollments,!events,!programOwners";
 
+    /**
+     * Query filter for attributes
+     */
     private String query;
 
-    private Set<String> attribute = new HashSet<>();
+    /**
+     * Comma separated list of attribute UIDs
+     */
+    private String attribute;
```

Comment:
What you can do in such a case to have this better described in OpenAPI is to annotate with 
```java
@OpenApi.Property( UID[].class, TrackerAttribute.class )
```
I assume this is about `TrackerAttribute`s. 
This would then document this input as being a "array" of tracker attribute UIDs.

Reply:
This is not really a collection of UIDs, right?
This is a collection of filters, that will be formed by a UID, operator and value, right?

I wouldn't fix it here as we have other stories to tackle the TrackedEntity controller but just want to be sure that we are on the same page. If this was a collection of UIDs we wouldn't need all the escaping process that is the core of this PR

Reply:
Not sure about this, [docs](https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-239/new-tracker.html#tracked-entities-get-apitrackertrackedentities) says this is a 
- Comma separated values of attribute UIDs
but in fact, looking at the code, we can actually apply filters